prep-html-skeleton/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Skeleton Exercise</title>
+  <body>
+    <h1>
+      I built a proper HTML Skeleton!.
+    </h1>
+  </body>
+</head>
```

Comment:
Hi Sixx, for your HTML skeleton, be sure to close the `</head>` element before you open your `<body>` element like this

```html
<head>
  <title>HTML Skeleton Exercise</title>
</head>
<body>
```